core/src/models/App.ts

```diff
@@ -24,19 +24,8 @@ import { AppOps } from "../modules/ops/app";
 import { LockableHelper } from "../modules/lockableHelper";
 import { ConfigWriter } from "../modules/configWriter";
 import { APIData } from "../modules/apiData";
-import { PluginOptionTypes } from "../classes/plugin";
 import { AppConfigurationObject } from "../classes/codeConfig";
 
-export interface AppOption {
```

Comment:
out of curiosity, what was the reason for moving this to the plugin vs on the app model?